### ISSUE TITLE
As a Service Developer I should be able to see the status of my service which relates to deploy #479

### ISSUE THREAD

**Issue by gevf** [gevf/test#5]
Functional architect/designer: @-mention
Technical architect: @-mention

Description
As a service developer I need to be able to access the status of my service.
Furter analysis is needed to identify what to offer on this page.

It could be interesting to include the following information:

possibility to compile service (dependent on issue #986)
compiling status (dependent on issue #986)
workflow status
Authorisation rules
Datamodel mapping
Shipment details
Translations
13.02.2019
Scoping this issue to include status information relevant for deploy.

The status of my service should contain information about the committed service to master. Might be a need to also have status of what is local in Studio as well.
Information that is relevant is:

compile status
UI components connected to datamodel
other?
The user shall from the status page be guided into the issues of the service in order to correct the service.

From where you deploy it shall be possible to get to the status page in order to see the status of the service.

Need/possible with auto-build?

Sketch (if relevant)

Status page which is relevant for deploy with possibility to grow over time - Should include statuses which indicates that the service could not be deployed due to the issue on the service, but also include status identifier that indicates that you service is not complete but could be deployed (will be relevant typically for texts)
Look at status for the service which is not commited, perhaps give information to the user that status could not be given as it is not commited or to display it at the same time? The last option might be difficult, so might need a status icon which tells the user that the code needs to be checked in
Navigation from/to (if relevant)
This page is available in the left side menu undr the "Publisere" area
It shall be possible to arrive here from the deploy area
It shall be possible to navigate further to the issue with the status

Technical considerations
Input (beyond tasks) on how the user story should be solved can be put here.

Acceptance criterea

Detailed description of functionality
Sketches
Tasks

  Analysis into what is needed for status of a service
  Sketches
  Auto save according to #404
  Create new issue for expanding the service status page
